src/number-form/index.html

```diff
@@ -49,7 +49,7 @@ <h1 id="title">WhatsApp Number</h1>
         </select>
         <input
           id="phone-number"
-          type=" number"
+          type="text"
```

Comment:
type="value"

Reply:
i think value will be good

Reply:
Sorry, I didn't get it... is value a valid term to use in type attribute? In this case I'm using text to apply the mask into phone number.